### ISSUE TITLE
Fails on systems without yum (but with dnf)

### ISSUE THREAD

**Issue by dfarrell07** [dfarrell07/ansible-opendaylight#19]
On distros that have moved to dnf as the default package manager over yum (like Fedora 22, Fedora 23), the role fails with a yum import error.

```
Traceback (most recent call last):
  File "/home/vagrant/.ansible/tmp/ansible-tmp-1453913472.43-6655968565982/yum", line 27, in <module>
    import yum
ImportError: No module named yum
```

The right way to fix this is to use the [`package`](https://docs.ansible.com/ansible/package_module.html) Ansible role, which supports both yum and dnf. That came out in Ansible 2.0, which was very recently released but hasn't yet been packaged and distributed as an RPM. The could be fixed by moving to the [`dnf`](http://docs.ansible.com/ansible/dnf_module.html) Ansible module, but that would break yum distros like CentOS 7. Could add some conditional to the tasks, but that's hack.

For now we'll target CentOS 7 over F22/F23 and move to the package module once it's out.

**Comment by dfarrell07** [comment]
CentOS now has Ansible 2.0.1.0-2.el7, so this is unblocked.

**Comment by dfarrell07** [comment]
This was fixed in #22 and e56dd8af2fb7bcff938cb2198d8127a64f366e9c.
